modules/ROOT/pages/syntax/keywords.adoc

```diff
@@ -0,0 +1,361 @@
+:description: This section contains a list of reserved keywords in Cypher.
+
+[[cypher-keywords]]
+= Keywords
+
+Keywords are words that have a special meaning in Cypher.
+
+The keywords are not recommended to be used as identifiers in the following contexts:
+
+* Variables
+* Labels
+* Relationship types
+
+If a snippet of Cypher can validly be a keyword as well as an unquoted identifier, it is always interpreted as a keyword.
+
+For instance in the following, `true` could be the variable `true` or the Boolean literal value `true`. Since `true` is a keyword, the query returns a single Boolean `true` value.
+
+.Query
+[source, cypher, role=test-result-skip]
+----
+WITH 123 AS true
+RETURN true AS x LIMIT 1
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| x
+
+| true
+
+1+d|Rows: 1
+|===
+
+If any keyword is quoted in backticks (```), such as `++`true`++`, it is always interpreted as an identifier in the given context.
+
+.Query
+[source, cypher, role=test-result-skip]
+----
+WITH 123 AS true
+RETURN `true` AS x LIMIT 1
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| x
+
+| 123
+
+1+d|Rows: 1
+|===
+
+
+== List of keywords
+
+The following lists of all keywords in Cypher.
```

Comment:
It would be nice with a test somewhere that fails when a new keyword is added and remind us to update this list.

Reply:
I think, I can come up with something like that. I will create a card.